substrate-sgx/sp-io/with_sgx.rs

```diff
@@ -488,57 +509,55 @@ pub mod crypto {
     }
 }
 
+ /// Interface that provides functions for hashing with different algorithms.
 pub mod hashing {
     use super::*;
-
+    /// Conduct a 256-bit Keccak hash.
     pub fn keccak_256(data: &[u8]) -> [u8; 32] {
-        warn!("hashing::keccak256 unimplemented");
-        [0u8; 32]
+        sp_core::hashing::keccak_256(data)
     }
 
+    /// Conduct a 512-bit Keccak hash.
+	pub fn keccak_512(data: &[u8]) -> [u8; 64] {
+		sp_core::hashing::keccak_512(data)
+	}
+
+
+    /// Conduct a 256-bit Sha2 hash.
     pub fn sha2_256(data: &[u8]) -> [u8; 32] {
         sp_core::hashing::sha2_256(data)
     }
 
+    /// Conduct a 128-bit Blake2 hash.
     pub fn blake2_128(data: &[u8]) -> [u8; 16] {
-        debug!("blake2_128 of {}", encode_hex(data));
```

Comment:
why would you delete debugging information? This can sometimes come handy to find out raw keys

Reply:
(it was us introducing these messages in the first place...)

Reply:
came in handy to just copy paste the substrate code while looking for possibel error causes. I'll redo it, just one moment.